src/main/java/io/openliberty/tools/intellij/lsp4jakarta/lsp4ij/codeAction/proposal/NewAnnotationProposal.java

```diff
@@ -0,0 +1,72 @@
+/*******************************************************************************
+ * Copyright (c) 2000, 2021 IBM Corporation and others.
```

Comment:
2023 copyright year, same for the other files

Reply:
👍